README.md

```diff
@@ -1,8 +1,9 @@
-:warning: This is an early version and may be unstable.
+# SBCF
```

Comment:
maybe put something in the readme about remembering to add to the changelog before you release?